backend/src/main/java/ai/verta/modeldb/authservice/PublicRoleServiceUtils.java

```diff
@@ -266,6 +268,18 @@ public WorkspaceDTO getWorkspaceDTOByWorkspaceId(
     return Collections.emptyList();
   }
 
+  @Override
+  public boolean createWorkspaceRoleBinding(
+          Long workspaceId,
+          WorkspaceType workspaceType,
+          String resourceId,
+          Long ownerId,
+          ModelDBServiceResourceTypes resourceType,
+          CollaboratorTypeEnum.CollaboratorType collaboratorType,
+          ProjectVisibility visibility) {
```

Comment:
Should we use the collaborator visibility?

Reply:
I changed it to use the common VisibilityEnum.Visibility type to make it more generic and maintain OSS compatibility.